### PR TITLE
UIEH-923: Providers, Packages, Titles filter - accordions do not have correct ARIA attributes

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
[UIEH-923](https://issues.folio.org/browse/UIEH-923): Providers, Packages, Titles filter - accordions do not have correct ARIA attributes

## Purpose
Element with a `tablist` role missed children with a `tab` role.

Related PR's

- Added additional props to `DefaultAcordionHeader` 
  https://github.com/folio-org/stripes-components/pull/1397

- Added headerProps property to `NotesSmartAccordion`
  https://github.com/folio-org/stripes-smart-components/pull/922

### CODE REVIEW

**Review by BogdanDenis — APPROVED** [review]

File: src/components/details-view/details-view.js [hunk 1]
```diff
@@ -331,6 +331,9 @@ class DetailsView extends Component {
                     id={listSectionId}
                     onToggle={onListToggle}
                     listType={listType}
+                    headerProps={{
+                      role: 'tab'
+                    }}
```
Comment (Vladyslav-Velytskyi) [inline src/components/details-view/details-view.js:336]: If you use one property in object, please write it in one string or use comma, for facilitate support code in future
```suggestion
                    headerProps={{ role: 'tab' }}
```
or
```suggestion
                    headerProps={{
                      role: 'tab',
                    }}
```

File: src/components/package/show/package-show.js [hunk 2]
```diff
@@ -324,6 +324,10 @@ class PackageShow extends Component {
       packageSelected,
     } = this.state;
 
+    const headerProps = {
+      role: 'tab'
```
Comment (Vladyslav-Velytskyi) [inline src/components/package/show/package-show.js:328]: Same here

File: src/components/provider/show/provider-show.js [hunk 3]
```diff
@@ -167,6 +167,9 @@ class ProviderShow extends Component {
     const hasProxy = hasIn('proxy.id', model);
     const hasToken = hasIn('providerToken.prompt', model);
     const hasProviderSettings = hasProxy || hasToken;
+    const headerProps = {
+      role: 'tab'
```
Comment (Vladyslav-Velytskyi) [inline src/components/provider/show/provider-show.js:171]: Same here

File: src/components/resource/resource-show.js [hunk 4]
```diff
@@ -256,6 +256,9 @@ class ResourceShow extends Component {
         type: 'success'
       });
     }
+    const headerProps = {
+      role: 'tab'
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/resource-show.js:260]: Same here

File: src/components/tags/tags-accordion.js [hunk 5]
```diff
@@ -38,6 +39,7 @@ class TagsAccordion extends React.Component {
       updateFolioTags,
       updateEntityTags,
       entityTags,
+      headerProps
```
Comment (Vladyslav-Velytskyi) [inline src/components/tags/tags-accordion.js:42]:
```suggestion
      headerProps,
```

File: src/components/title/show/title-show.js [hunk 6]
```diff
@@ -215,6 +215,9 @@ class TitleShow extends Component {
     };
     let submit;
     const submitAddToCustomPackage = (event) => submit(event);
+    const headerProps = {
+      role: 'tab'
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/show/title-show.js:219]: Same here

**Review by Vladyslav-Velytskyi — CHANGES_REQUESTED** [review]

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Pull request merged**